src/components/shared/forms/pillContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Pill } from './pill';
+import { Link } from 'utilities';
+import './styles/pillContainer.css';
+
+export const PillContainer = (props) => {
+  const { pills, onSvgClick, svg } = props;
+
+  return (
+    <div className="pill-container">
+      {pills.value.map((pill, idx) => (
+        <Pill
+          key={idx}
+          label={pill}
+          svg={svg}
+          onSvgClick={onSvgClick(idx)}
+        ></Pill>
+      ))}
+    </div>
+  );
+};
+
+PillContainer.propTypes = {
+  svg: PropTypes.string,
+  onSvgClick: PropTypes.func,
+  pills: PropTypes.instanceOf(Link)
```

Comment:
Instead of passing the `Link` to the `Pills` component, it would be better to pass an array. Otherwise, it isn't as reusable.

Reply:
Changed.